### DISCUSSION TITLE
How to change the line "TODO: 1 of 1 tasks shown"

### DISCUSSION THREAD

**Discussion by blogdemoi** [todotxt/todo.txt-cli#406] [Q&A] [answered]
Hello,

I am using todo.txt cli with conky following this tutorial: https://github-wiki-see.page/m/todotxt/todo.txt-cli/wiki/Linux-with-Conky and I would like to change the following line (thats shows on my Linux Desktop) "TODO: 1 of 1 tasks shown" to the French language so that it read: "TODO: 1 de 1 taches montrées".

![LyayO1D](https://user-images.githubusercontent.com/83965802/222502604-2927f055-8f71-4d5b-a311-79fe5cb0f0e4.png)

In which file do I have to change that?

Thank you.

**Comment by inkarkat** [comment, marked as answer]
Unfortunately, there isn't yet a localization mechanism built into todo.txt-cli, so all messages are hard-coded inside the script itself.

The output you're interested in is in function `_list()`, [here](https://github.com/inkarkat/todo.txt-cli/blob/2937a8b3166759df4a9e59b7351522256135bdba/todo.sh#L894).

I wouldn't recommend editing that file, you'll have to redo that (or patch / rebase it) each time there's a new todo.txt-cli release.

For Conky, I would simply omit the summary at the end by setting `TODOTXT_VERBOSE=0`; that'll also save you some valuable space. You may even want to limit the amount of tasks shown to a maximum (e.g. of 10); for that, you'll likely pipe `todo.sh`'s output into `sed` or `awk`, and you easily can do the counting there.

Apropos piping; Conky may interpret some characters (`#` and `$`), which can lead to funny output if your tasks have those, too. Escaping those is easy, though; I've written a tiny [conkyEscape](https://github.com/inkarkat/conky/blob/master/bin/conkyEscape) sed script for that.

**Reply by blogdemoi** [reply]
Thank you.

**Reply by blogdemoi** [reply]
> For Conky, I would simply omit the summary at the end by setting TODOTXT_VERBOSE=0

Where do you edit this? I tried in todo.sh but with no results. Thank you.

**Reply by inkarkat** [reply]
It should never be necessary to edit `todo.sh` itself. The configuration consists of plain environment variables, so you can put them into `todo.cfg` (there are various locations where this could be stored). You could also simply `export TODOTXT_VERBOSE=0` (e.g. in your `~/.bashrc`, but that isn't as clean as using the dedicated config file), or if you just want to set this for the Conky invocation but keep the default verbosity when you invoke it yourself, just prepend the value: `TODOTXT_VERBOSE=0 todo.sh list ...`

**Reply by blogdemoi** [reply]
Thank you, it worked perfectly. I can now have my todos on my laptop and phone, synchronized via a Nextcloud server. What a simple but useful tool!
